script.js

```diff
@@ -34,3 +35,234 @@ link2.addEventListener('click', () => {
+    </ul>
+    <div class="btn-container">
+        <button class="btn-3 open">See Project</button>
+    </div>
+    </div>
+`;
+
+const obj = [{
+  name: ['Multi-Post Stories', 'Keeping Track of Hundreds of components website'],
+  description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"],
+  img: 'images/project-background.png',
+  technologies: ['html', 'bootstrap', 'Ruby on Rails'],
+  live: 'https://amrit131.github.io/portfolio-mobile/',
+  source: 'https://github.com/Amrit131/portfolio-mobile',
+}];
+
+const open = document.getElementsByClassName('open');
+for (let i = 0; i < open.length; i++) {
+  open[i].addEventListener('click', () => {
+    const main = document.createElement('div');
+    main.className = 'main';
+    const popup = document.createElement('div');
+    popup.className = 'pop-up';
+    popup.innerHTML = `
+          <div class="gymfit-container">
+          <div class="post-container">
+              <h3 class= "header-3">${obj[0].name[0]}</h3>
+              <h3 class= header3>${obj[0].name[1]}</h3>
+              <a class= "close-btn" href= "#">
+                  <span class= "close-bar" id= "bar1"></span>
+                  <span class= "close-bar" id= "bar2"></span>
+              </a>
+              <ul id="tech-btn1">
+                  <li class="bttn"><button>${obj[0].technologies[0]}</button></li>
+                  <li class="bttn" ><button>${obj[0].technologies[1]}</button></li>
+                  <li class="bttn" ><button>${obj[0].technologies[2]}</button></li>
+              </ul>
+              <div class="pop-up-desktop">
+                  <img class="gymfit-img" src="${obj[0].img}" alt="Gym Fit Image">
+                  <div class = "para-button">
+                      <p class="stories-para">${obj[0].description[0]}</p>
+                      <p class="stories-para1">${obj[0].description[1]}</p>
+                      <div class="btn-container1">
+                          <a href= ${obj[0].live} ><button class= "close" id="btn1" type= "button">See Live <img id= "icon-img" src= "images/see-live-icon.png"></button></a>
+                          <a href= ${obj[0].source} ><button class= "close" id="btn2" type= "button">Source code <img id= "icon-img" src= "images/source-icon.png"></button></a>
+                      </div>
+                  </div>
+              </div>
+          </div>
```

Comment:
- There are some elements overflowing in your popup window, check the image referenced below 👇. Please fix this issue and be sure that all elements are displayed within the popup window 👍  👍

Your project (Pay attention to the bottom part, the images and button are overflowing):
<img width="915" alt="popup window" src="https://user-images.githubusercontent.com/73607512/196972841-c53d3552-75b7-4b89-adb0-5745f661472d.PNG">

Reply:
I have implemented the given feedback.
![mobile](https://user-images.githubusercontent.com/87594100/197000681-88f56075-3489-4028-ae72-a232c9b8b028.png)
![Tab](https://user-images.githubusercontent.com/87594100/197000693-b9fd50a9-06ca-4fa2-8260-08fd3eeafb2f.png)
![Tab-1](https://user-images.githubusercontent.com/87594100/197000702-6a1d372b-704b-466d-9d5e-fc104ff2e6e1.png)